src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = u('.js-Slideshow-slides'),
+    slides = u('.js-Slideshow-slide'),
+    nextTrigger = u('.js-Slideshow-next'),
+    prevTrigger = u('.js-Slideshow-prev'),
+    currentCountElement = u('.js-Slideshow-current'),
+    totalCountElement = u('.js-Slideshow-total'),
```

Comment:
Is there something additional that needs to be done to keep these within the `element` scope, or does that happen automagically? Like, previously they were `element.querySelector()` vs `document.querySelector()`.